next.config.js

```diff
@@ -1,21 +1,24 @@
 const { fetchZestyRedirects } = require('./src/lib/zesty/fetchRedirects');
 
 module.exports = {
-	trailingSlash: true,
-	async redirects() {
-		return await fetchZestyRedirects();
-	},
-	env: {
-		zesty: {
-			instance_zuid: '8-aaeffee09b-7w6v22',
-			stage: 'https://kfg6bckb-dev.webengine.zesty.io',
-			production: 'https://www.zesty.io',
-			stage_password: '',
-			auth: '',
-			src_dir: '/src'
-		},
-		eslint: {
-			ignoreDuringBuilds: true
-		}
-	}
+  images: {
```

Comment:
@ardeay 

This is required in order for next/image to work you can define here allowed domains to optimize

more details on that can be found on docs -> https://nextjs.org/docs/basic-features/image-optimization